docs/appendix/document-creator.md

```diff
@@ -405,3 +405,72 @@ _Note: **Please replace all the placeholder "<...>" with your values.**_
 _Note: **The "documentStorage" field is optional, if you do not have any document storage endpoint setup, you can omit this entire section.**._
 
 ---
+
+## Generic Template (Bill of Lading)
+
+We have designed and created a simple generic template (Bill of Lading) config file for you to use.
+
+Click <a href="/docs/appendix/document-creator/bill-of-lading/ebl-generic-template-config-file.json" target="_blank" rel="noopener noreferrer" download="ebl-generic-template-config-file.json">here</a> to download a config file.
+
+![Generic Template](/docs/appendix/document-creator/bill-of-lading/ebl-generic-template.png)
+
```

Comment:
```suggestion
![Generic Template - Bill of Lading](/docs/appendix/document-creator/bill-of-lading/ebl-generic-template.png)

```

also, able to not use govtech logo for ebl? maybe use the TT one will do

Reply:
i have changed it already.